scripts/drag-n-bar.js

```diff
@@ -1377,18 +1377,25 @@ H5P.DragNBar.prototype.findNewPoint = function (originX, originY, angle, distanc
     };
 
     // set start angle
+    let angle = 0;
     const angleTransformList = $element[0].style.transform.split("rotate(");
-    const angleTransform = parseInt(angleTransformList[1]);
-    let angle = angleTransform;
+    if(angleTransformList != "") {
+    angleTransform = parseInt(angleTransformList[1]);
+    angle = angleTransform;
     if(angle == 0  || angle == null || angle == undefined) {
       angle = 0;
     }
     frame.rotate = angle;
+  }
 
     // set start transform
     const transformCSSTranslateXYArray = $element[0].style.transform.split("px");
-    const transformCSSTranslateX = parseInt(transformCSSTranslateXYArray[0].match(/-?\d+/g)[0]);
-    const transformCSSTranslateY = parseInt(transformCSSTranslateXYArray[1].match(/-?\d+/g)[0]);
+    let transformCSSTranslateX = 0;
+    let transformCSSTranslateY = 0;
+    if(transformCSSTranslateXYArray[0] != "") {
```

Comment:
Unless we're checking for `null` and `undefined`, we should use triple equals in JS

```suggestion
    if(transformCSSTranslateXYArray[0] !== "") {
```